.github/workflows/deploy-prod.yml

```
@@ -84,11 +94,24 @@ jobs:
+      - name: Upload Wrangler logs (Admin)
+        if: failure()
+        continue-on-error: true
+        uses: actions/upload-artifact@v4
         with:
-          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
-          packageManager: bun
-          # Avoid Cloudflare "Invalid commit message" (8000111) for long UTF-8 git messages (workers-sdk#11749).
-          command: pages deploy admin/dist --project-name=zedi-admin --commit-message "Deploy admin from GitHub Actions (main) ${{ github.sha }}"
+          name: wrangler-logs-admin
+          path: ${{ runner.home }}/.config/.wrangler/logs/
```

actionlint (1.7.11)

[error] property "home" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)